src/Electron/electron.ts

```diff
@@ -371,7 +371,16 @@ export class ElectronContainer extends WebContainerBase {
     }
 
     public showNotification(title: string, options?: NotificationOptions) {
-        throw new TypeError("showNotification requires an implementation.");
+        const Notification = this.electron.Notification;
+        const notify = new Notification(Object.assign(options || {}, { title: title }));
+        if (options["onClick"]) {
```

Comment:
if both onClick and notification are set, won't this result in two subscriptions?

Reply:
Yes, but I felt that if you went out of your way to work that out then you want both subscribed.  Note that the notification object is provided on the default w3c polyfill that redirects to showNotification which is in place for windows 7.  With windows 10 the polyfill should be turned off and I'll be working on surfacing this better in the future.

Reply:
Also, on a similar topic, won't having show and once called cause an issue?

Reply:
The 'show' event can be fired multiple times so I only wanted it attached once.

https://electronjs.org/docs/api/notification#event-show